cypress.json

```diff
@@ -7,7 +7,14 @@
         "pass": "Dog8code",
         "git_user": "",
         "git_password": "",
-        "tackleUrl": "https://tackle-konveyor-tackle.apps.mig01.cnv-qe.rhcloud.com/"
+        "tackleUrl": "https://tackle-konveyor-tackle.apps.mig01.cnv-qe.rhcloud.com/",
+        "jobFunctionName": "upgrade-jobFunction",
+        "stakeHolderGroupName": "upgrade-stakeHolderGroup",
```

Comment:
@ibragins This is not the correct place for hardcoding data name. You can hardcode them in the test file itself but not at cypress.json .
You can also create an upgrade.json under fixtures and use it .

Reply:
Removed and used fixture instead